docs/data/migration/migration-charts-v7/migration-charts-v7.md

```diff
@@ -118,7 +118,29 @@ The impacted properties are:
 - The `xAxisKey`, `yAxisKey`, and `zAxisKey` have been renamed `xAxisId`, `yAxisId`, and `zAxisId`.
 - The `highlighted` and `faded` properties of `series.highlightScope` have been renamed `highlight` and `fade`.
 
-## Legend props propagation ✅
+## Legend modification
+
+### Legend is now an HTML element
+
+The `ChartsLegend` is now an HTML component.
+This make it simpler to customize but prevent from using the `ChartContainer`.
+
+It must be inside the Data Provider to get the data, but outside the Surface since it's not an SVG element.
```

Comment:
```suggestion
It must be rendered inside the `ChartDataProvider` to get the data, but outside the `ChartsSurface` since it's not an SVG element.
```

Reply:
I avoid the `ChartDataProvider` because we also have the `ChartDataProviderPro` and `RadarDataProvider` so I prefered the "Data Provider" to say "Pick the one that feet you need"